build/ublue-os-just/40-nvidia.just

```diff
@@ -59,15 +59,44 @@ configure-nvidia ACTION="prompt":
     fi
 
 # Switch between Nvidia image and NVK
-toggle-nvk:
+toggle-nvidia ACTION="prompt":
     #!/usr/bin/bash
     CURRENT_IMAGE=$(rpm-ostree status -b --json | jq -r '.deployments[0]."container-image-reference"' | sed -E 's/^.+\/(.+:.+)$/\1/')
     CURRENT_URI=$(rpm-ostree status -b --json | jq -r '.deployments[0]."container-image-reference"' | sed -E 's/^(.+\/).+:.+$/\1/')
     NEW_IMAGE=$CURRENT_IMAGE
     if [[ "$CURRENT_IMAGE" =~ "nvidia" ]]; then
-      NEW_IMAGE=${CURRENT_IMAGE/-nvidia/}
+      IS_CURRENTLY_NVIDIA=true
+      echo "Current nvidia driver: Nvidia (From Nvidia Inc, Proprietary)"
     else
-      NEW_IMAGE=${CURRENT_IMAGE/:/-nvidia:}
+      IS_CURRENTLY_NVIDIA=FALSE
+      echo "Current nvidia driver: NVK (Open Source, MESA)"
+    fi
+    OPTION={{ ACTION }}
+    if [ "$OPTION" == "prompt" ]; then
+      echo "${bold}Enable or Disable Nvidia driver${normal}"
+      echo 'What would you like to do?'
+      OPTION=$(ugum choose "Enable" "Disable")
+    elif [ "$OPTION" == "help" ]; then
+      echo "Usage: ujust toggle-nvidia <option>"
+      echo "  <option>: Specify the quick option - 'enable' or 'disable'"
+      echo "  Use 'enable' to Enable the proprietary Nvidia driver from Nvidia Inc (reboot required)."
+      echo "  Use 'disable' to Enable the open source Nvidia driver (reboot required)."
+      exit 0
+    fi
+    if [ "$OPTION" == "Enable" ] || [ "${OPTION,,}" == "enable" ]; then
+      if $IS_CURRENTLY_NVIDIA; then
```

Comment:
this and line 94 fails as it needs to be `if [ "$IS_CURRENTLY_NVIDIA" == "true" ]; then`
and false for line 94 :)